DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateCoursesController.cs

```diff
@@ -0,0 +1,101 @@
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Helpers.FilterOptions;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+
+    [FeatureGate(FeatureFlags.RefactoredTrackingSystem)]
+    [Authorize(Policy = CustomPolicies.UserCentreAdmin)]
+    [SetDlsSubApplication(nameof(DlsSubApplication.TrackingSystem))]
+    [SetSelectedTab(nameof(NavMenuTab.Delegates))]
+    [Route("TrackingSystem/Delegates/Courses")]
+    public class DelegateCoursesController : Controller
+    {
+        private const string CourseFilterCookieName = "DelegateCoursesFilter";
+        private readonly ICourseCategoriesDataService courseCategoriesDataService;
+        private readonly ICourseService courseService;
+        private readonly ICourseTopicsDataService courseTopicsDataService;
+
+        public DelegateCoursesController(
+            ICourseService courseService,
+            ICourseCategoriesDataService courseCategoriesDataService,
+            ICourseTopicsDataService courseTopicsDataService
+        )
+        {
+            this.courseService = courseService;
+            this.courseCategoriesDataService = courseCategoriesDataService;
+            this.courseTopicsDataService = courseTopicsDataService;
+        }
+
+        [Route("{page=1:int}")]
+        public IActionResult Index(
+            string? searchString = null,
+            string? sortBy = null,
+            string sortDirection = GenericSortingHelper.Ascending,
+            string? filterBy = null,
+            string? filterValue = null,
+            int page = 1,
+            int? itemsPerPage = null
+        )
+        {
+            sortBy ??= DefaultSortByOptions.Name.PropertyName;
+            filterBy = FilteringHelper.GetFilterBy(
+                filterBy,
+                filterValue,
+                Request,
+                CourseFilterCookieName,
+                CourseStatusFilterOptions.IsActive.FilterValue
+            );
+
+            var centreId = User.GetCentreId();
+            var categoryId = User.GetAdminCourseCategoryFilter();
+            var centreCourses =
+                courseService.GetCentreSpecificCourseStatisticsWithAdminFieldResponseCounts(centreId, categoryId);
+            var categories = courseCategoriesDataService.GetCategoriesForCentreAndCentrallyManagedCourses(centreId)
+                .Select(c => c.CategoryName);
+            var topics = courseTopicsDataService.GetCourseTopicsAvailableAtCentre(centreId).Select(c => c.CourseTopic);
```

Comment:
Seeing as this is identical to the CourseSetup controller's logic, we'd probably want to reuse it there. So I'd imagine it would instead need to be a new public method in the courseService.